ResourceStatusSystem/Client/ResourceManagementClient.py

```diff
@@ -219,7 +219,7 @@ def addOrModifyGGUSTicketsCache( self, gocSite = None, link = None, openTickets
   def selectDowntimeCache( self, downtimeID = None, element = None, name = None,
                            startDate = None, endDate = None, severity = None,
                            description = None, link = None, dateEffective = None,
-                           lastCheckTime = None, gocdbServiceType = None, meta = None ):
+                           lastCheckTime = None, gOCDBServiceType = None, meta = None ):
```

Comment:
@chaen That's true, I just didn't want to break the convention of the first lowercase letter for every parameter :P